tests/bitswap_stress_test.rs

```diff
@@ -0,0 +1,48 @@
+use ipfs::{Block, Node};
+use libipld::cid::{Cid, Codec};
+use multihash::Sha2_256;
+
+fn filter(i: usize) -> bool {
+    i % 2 == 0
+}
+
+#[async_std::test]
+async fn bitswap_stress_test() {
```

Comment:
could you add a comment or something here, to describe how does this in fact stress the bitswap, maybe what kind of tracing are you expected to see and so on?

Reply:
ah yeah, good point; it can be `#[ignore]`d in general, it's for manual debugging